components/ItemCard.js

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+
+export default function ItemCard({ name, date, quantity }) {
+  return (
+    <StyledItem>
+      <StyledName>{name}</StyledName>
+      <StyledQty>Qty:</StyledQty>
+      <StyledQtyValue>{quantity}</StyledQtyValue>
+      <StyledDate>date:</StyledDate>
+      <StyledDateValue>{date}</StyledDateValue>
+    </StyledItem>
+  );
+}
+
+const StyledItem = styled.li`
+  border: 1px solid black;
+  margin: 1%;
+  margin-left: 0;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: 'name . .' 'name qty date' 'name qtyvalue datevalue  ';
+  padding: 15px;
+`;
+const StyledName = styled.h3`
+  margin: 2%;
+  justify-self: start;
+  align-self: center;
+  grid-area: name;
+  display: grid;
```

Comment:
bin mir da nicht mehr sicher, aber kann das vielleicht raus? Wird die Eigenschaft grid nicht an die Kinder weitergegeben? Wenn nicht - feel free to ignore :)